README.MD bug fixes (#47041)
Azure/azure-sdk-for-net: sdk/cloudmachine/README.MD

```dotnetcli
 (✓) Done: Resource group: cm125957681369428 (627ms)
```
And if you go to your Azure portal, or execute the following az command, you can see the resource group created. The resource group will contain resources such as Storage, ServiceBus, and EventGrid. 
```dotnetcli
az resource list --resource-group <resource_group_from_command_line> --output table
```

#### Use CDK to add resources to the CloudMachine

Since we are writing an AI application, we need to provision Azure OpenAI resources. To do this, add the follwoing class to the end of the `Program.cs` file:
```csharp
class AssistantService {
    internal static void Configure(CloudMachineInfrastructure cm) {
        cm.AddFeature(new OpenAIFeature() { Chat = new AIModel("gpt-4o-mini", "2024-07-18") });
    }
}
```

> And then change the configuration call at the begining of the file to:

Reviewer: ```suggestion
Now change the configuration call at the beginning of the file to:
```

    Task<string> Chat(string message);
}
```

> You can now start the application and browse to the /chat endpoint [TDB on how to pass the parameter]

Reviewer: ```suggestion
You can now start the application and browse to the /chat endpoint [TBD on how to pass the parameter]
```
